### PR TITLE
using specified stacktrace when given an error object

### PR DESCRIPTION
**Pull request by keithhalterman**

### CODE REVIEW

File: src/auto-trace.js [hunk 1]
```diff
@@ -100,7 +100,9 @@ export function withoutStacktrace(rawError, ...extraMiddlewares){
  */
 function wrapObjectWithError(err, stacktraceErr) {
```
Comment (keithhalterman) [inline src/auto-trace.js]: Wondering if there's a good way to write jasmine tests for a private method?
(Hopefully something other than separate production/dev builds)

Comment (blittle) [inline src/auto-trace.js]: Maybe just pull the private function out into a helper file. Then test the helper?

Comment (keithhalterman) [inline src/auto-trace.js]: Great plan! Thanks!

**Comment by blittle** [comment]
:camel: good work!

**Pull request merged**